_posts/en/2021-08-18-user-engagement-in-service-delivery.md

```diff
@@ -0,0 +1,134 @@
+What has *not* changed is that when they *do* interact with service delivery, they want a **great** user experience.
+
+[GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html) applies the mindset of providing public services to Canadians that are "simple to use".
+To design "user-focused" products, the user should be involved in every aspect of the research to optimize the time and effort put into understanding its needs and problems.
+
+We must design better products for citizens and public servant colleagues.
+*Why?* Because "we" demand it.
+Because a long-lasting, well-structured relationships is the secret ingredient to make users happy.
+
+### Creating personas
+
+In the IT Strategy team, we designed and implemented some products and luckily, we had a chance to involve users throughout the design process as well as test our prototypes before launching.
+Our first step was learning about users, their context, their objectives, and obstacles to inform our general direction.
+We created personas to help us get to know our customers, their behaviour, goals, frustrations, to understand who we are designing for, and if it meets the needs of our target audience.
+
+This is what happened during the [DevOps Self-Assessment tool](https://sara-sabr.github.io/ITStrategy/devops-self-assessment.html) design sprint.
+Based on research, we created fictional characters to help us understand the users we are designing for.
+We discovered different types of users with their behaviour patterns, goals, skills, attitudes, and background information, as well as the environment in which they operate to see where we should focus and improve while designing our application.
+The main user of the tool is a public servant in IITB looking to continuously improve and support the efforts of its peers.
+By understanding the expectations, concerns, and motivations of our target users, we came up with a design-focused solution.
+To design for a public servant while being public servants ourselves, it is so easy to create a self-referential product as if we are making the product only for us, when in fact the target audience may be quite unlike us.
+We also created a mock-up which was recently presented to a real group of users in order to collect feedback for further improvement.
+
+Thinking about the needs of our fictional – end user persona, we tried to shape the product strategy and create the best mock-up for *him*.
+It is worth mentioning that although user personas helped us to prioritize the features, it was not the only tool used for prioritization.
+We were constantly collecting feedback from stakeholders to keep a proper balance between the needs of the business and the needs of the users, in order to create a solution that would satisfy both.
+With the feedback received, we tried to iterate the overall experience for the user.
```

Comment:
I forgot to leave a comment on this. Not sure "iterate" is the right verb here, but I understand the idea behind it. Maybe it needs rephrasing.